terraform/services/backup-plan/primary.tf

```diff
@@ -0,0 +1,211 @@
+}
+
+data "aws_iam_policy_document" "primary_backup_policy" {
+  for_each = toset(local.apps)
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "AWS"
+      identifiers = [data.aws_caller_identity.current.account_id]
+    }
+
+    actions = [
+      "backup:CopyIntoBackupVault",
+    ]
+
+    resources = [aws_backup_vault.primary_backup_vault[each.value].arn]
+  }
+}
+
+resource "aws_backup_vault_policy" "primary_backup_vault_policy" {
+  for_each          = toset(local.apps)
+  backup_vault_name = aws_backup_vault.primary_backup_vault[each.value].name
+  policy            = data.aws_iam_policy_document.primary_backup_policy[each.value].json
+}
+
+resource "aws_backup_plan" "aws_backup_plan" {
+  for_each = toset(local.apps)
+  name     = "cdap_managed_backup_plan_${each.value}"
+  #only the 4hr rule should be copied to secondary
+  rule {
+    rule_name         = "4Hourly_1"
+    target_vault_name = aws_backup_vault.primary_backup_vault[each.value].name
+    schedule          = "cron(0 */4 * * ? *)"
+    start_window      = 60
+    completion_window = 180
+    copy_action {
+      destination_vault_arn = aws_backup_vault.secondary_backup_vault[each.value].arn
+    }
+
+    lifecycle {
+      delete_after = 1
+    }
+  }
+
+  rule {
+    rule_name         = "Daily_7"
+    target_vault_name = aws_backup_vault.primary_backup_vault[each.value].name
+    schedule          = "cron(0 4 * * ? *)"
+    start_window      = 60
+    completion_window = 180
+
+    lifecycle {
+      delete_after = 7
+    }
+  }
+
+  rule {
+    rule_name         = "Weekly_35"
+    target_vault_name = aws_backup_vault.primary_backup_vault[each.value].name
+    schedule          = "cron(0 0 ? * SAT *)"
+    start_window      = 60
+    completion_window = 180
+
+    lifecycle {
+      delete_after = 35
+    }
+  }
+
+  rule {
+    rule_name         = "Monthly_90"
+    target_vault_name = aws_backup_vault.primary_backup_vault[each.value].name
+    schedule          = "cron(0 0 1 * ? *)"
+    start_window      = 60
+    completion_window = 180
+
+    lifecycle {
+      delete_after = 90
+    }
+  }
+
+  depends_on = [
+    aws_backup_vault.secondary_backup_vault
+  ]
+
+  tags = {
+    cms-cloud-service = "AWS Backup"
+    Backup_Schedule   = "4hr1_d7_w35_m90"
+  }
+}
+
+resource "aws_backup_selection" "aws_backup_selection" {
+  for_each = toset(local.apps)
+  # This iam role is the one CMS is using for their backup plan.
+  iam_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/delegatedadmin/developer/cms-oit-aws-backup-service-role"
+  name         = "cdap_managed_backup_selection_${each.value}"
+  plan_id      = aws_backup_plan.aws_backup_plan[each.value].id
+  resources = [lower("arn:aws:rds:us-east-1:${data.aws_caller_identity.current.account_id}:cluster:${each.value}-${var.env}")]
+
+  selection_tag {
+    type  = "STRINGEQUALS"
+    key   = "Backup"
+    value = "cdap-managed"
+  }
+}
```

Comment:
The backup selection must select backups by tag as required by CMS Cloud governance (see https://cloud.cms.gov/creating-custom-aws-backup-plans).

The resources attribute must be removed, and the selection_tag block must set the key as `AWS_Backup` and the value as `4Hours1CA_Daily7_Weekly35_Monthly90`.